src/main.cc

```diff
@@ -253,6 +253,7 @@ int main(int argc, char *argv[])
         exitCode = app->exec();
     }
 
+    app->_shutdown();
```

Comment:
By this point the event loop is gone too. We should probably call [sendPostedEvents](http://doc.qt.io/Qt-5/qcoreapplication.html#sendPostedEvents) to destroy any stragglers after invoking _shutdown.